plugins/MumblePlugin.h

```diff
@@ -1518,6 +1519,35 @@ struct MUMBLE_API_STRUCT_NAME {
 
 	// -------- Request functions --------
 
+#	if SELECTED_API_VERSION >= MUMBLE_PLUGIN_VERSION_CHECK(1, 3, 0)
+
+	/**
+	 *
+	 * Requests Mumble to link the given channel to the given linkID.
+	 *
+	 * @param callerID The ID of the plugin calling this function
+	 * @param channelID The ID of the channel to link
+	 * @param linkID The ID of the base channel to link to
+	 * @returns The error code. If everything went well, STATUS_OK will be returned.
+	 */
+	mumble_error_t(MUMBLE_PLUGIN_CALLING_CONVENTION *requestChannelLink)(mumble_plugin_id_t callerID,
```

Comment:
I'm wondering whether putting the verb before the subject in the function name would make things more clear. E.g. `requestLinkChannels` 
That would seem more natural to me - what do you think?
(The same would of course also apply to the unlink function)

Reply:
While writing this implementation, I thought that the link was another concept. But  when my implementation is done., I realize that link ID is also a channel. That's why I put names like that. In a nutshell, renaming as requestLinkChannels is more meaningful. So I will update